Podcast/SearchITunesViewController.swift

```diff
@@ -13,10 +13,16 @@ class SearchITunesViewController: ViewController, UITableViewDelegate, UITableVi
     
     var searchController: UISearchController!
     var tableView: EmptyStateTableView!
+    var initialQuery: String?
     
     var searchResults: [Series] = []
     let seriesCellIdentifier = "SeriesCell"
     let seriesCellHeight: CGFloat = 95
+
+    convenience init(query: String?) {
```

Comment:
since this is a convenience init let's have query not be optional. If someone wanted to initailize this viewcontroller without a query they could just use the regular init